src/applications/static-pages/sass/modules/_m-facilities.scss

```diff
@@ -364,4 +364,10 @@ div.field-vet-center-feature-content  {
   li {
     margin-bottom: 0px
   }
+}
+
+.h4-vet-center {
+  font-size: 16px;
+  font-family: "Source Sans Pro";
+  font-weight: 700;
```

Comment:
Can we use `force-small-header` here instead?

We can move it outside of `facilities_health_service` if need be.

Reply:
yeah, didn't realize values are the same